mkosi/sandbox/__init__.py

```diff
@@ -619,7 +619,8 @@ def main() -> None:
     upperdir = ""
     workdir = ""
     chdir = None
-    become_root = suppress_chown = unshare_net = unshare_ipc = False
+    become_root = unshare_net = unshare_ipc = False
+    suppress_chown = bool(os.getenv("MKOSI_CHROOT_SUPPRESS_CHOWN", False))
```

Comment:
This should be processed outside of mkosi-sandbox. mkosi should look for the environment variable and pass --suppress-chown to chroot_cmd() when running scripts if it is set.